ui/ts/components/table.ts

```diff
@@ -133,7 +133,7 @@ module Components {
           m("th",
             {
               onclick: (e: any): void => this.SetSortColumn(col),
-              className: this.IsSortColumn(col) ? sortClass : null
+              className: this.IsSortColumn(col) ? sortClass : ""
```

Comment:
it was actually assigning `class="null"` to the html elements. With this, the class is actually blank.